.env-template

```diff
@@ -31,3 +31,16 @@ SENTRY_AUTH_TOKEN=
 
 # FACEBOOK
 NEXT_PUBLIC_FACEBOOK_APP_ID=
+
+# NEAR
+# TESTNET
+NETWORK_ID=
```

Comment:
i'm suggest to use prefix identifier, e.g `NEAR_NETWORK_ID`

BTW can we use this configuration from database ? @ical10 @abdulhakim2902